pkg/api/crio/crio.go

```diff
@@ -419,6 +421,16 @@ func removeAllContainers() {
 	log.Debug().Msgf("Finished removing all Buildah containers.")
 }
 
+func removeContainer(containerID string) error {
```

Comment:
this should use /cedana/bin no?

Reply:
Yeah it's the first in the path for systemd service PATH variable